migrations/sql/V2019.02.26.9.21__alter_party_table_rename_address_columns.sql

```diff
@@ -0,0 +1,46 @@
+ALTER TABLE party
+    ADD COLUMN address_type_code varchar(3) DEFAULT 'CAN' NOT NULL;
+
+ALTER TABLE party RENAME province_code TO region_code;
+ALTER TABLE party RENAME postal_code   TO post_code;
+ALTER TABLE party
+    RENAME CONSTRAINT "party_province_code_fkey" TO "party_region_code_fkey";
+
+CREATE TABLE IF NOT EXISTS address_type_code (
+    address_type_code character varying(3)                   NOT NULL,
+    description       character varying(50)                  NOT NULL,
+    display_order     integer                                        ,
+    active_ind        boolean                  DEFAULT true  NOT NULL,
+    create_user       character varying(60)                  NOT NULL,
+    create_timestamp  timestamp with time zone DEFAULT now() NOT NULL,
+    update_user       character varying(60)                  NOT NULL,
+    update_timestamp  timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE address_type_code OWNER TO mds;
+COMMENT ON TABLE address_type_code IS 'Lookup table for valid address types';
+
+
+INSERT INTO address_type_code
+    (
+    address_type_code,
+    description,
+    display_order,
+    create_user,
+    update_user
+    )
+VALUES
+    ('CAN', 'Canada', 10, 'system-mds', 'system-mds'),
+    ('USA', 'United States', 20, 'system-mds', 'system-mds'),
+    ('INT', 'International', 30, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+ALTER TABLE province_code RENAME TO region_code;
```

Comment:
We might want to consider using a different name for this table... "region" has a very specific meaning in EMPR (and the rest of Natural Resource Ministries). I had suggested "sub_division_code" which would in essence be a child of a table called "country" (at some point maybe)... but it's a generic term used to describe things like states, provinces, territories, etc.

Reply:
I can rename the db table, yeah. I'm really trying to just go with consensus on these, so can we make sure there's a proper consensus if I go through another rename? @nayr974 

I think region_code still makes sense at the API level, though. Since it'll be in an object, in an array that's labelled `addresses`

Thanks!